src/app/services/environments.service.ts

```diff
@@ -643,4 +643,44 @@ export class EnvironmentsService {
   public setEnvironmentCORSHeaders() {
     this.eventsService.injectHeaders.emit({ target: 'environmentHeaders', headers: CORSHeaders });
   }
+
+  /**
+   * Create a route based on a environment log entry
+   */
+  public createRouteFromLog(logUUID?: string) {
+    const environmentsLogs = this.store.get('environmentsLogs');
+    const uuidEnvironment = this.store.get('activeEnvironmentUUID');
+    const log = environmentsLogs[uuidEnvironment].find(environmentLog => environmentLog.uuid === logUUID)
```

Comment:
add missing semicolon (please run the linter before pushing :) )

Reply:
I really forgot to run the linter, sorry. But it's fixed.